README.md

```diff
@@ -51,10 +51,12 @@ The [bash](https://www.gnu.org/software/bash) shell is available on all commonly
 
 ### Install Python package
 
-If you wish to use our programs, perhaps without developing them further, run the command<br/>
-`pip install tiatoolbox`<br/>
-To understand better how the programs work, study the jupyter notebooks referred to under the heading **Examples Taster**. For more information, you may also run<br/>
-`tiatoolbox --help`<br/>
+If you wish to use our programs, perhaps without developing them further, run the command `pip install tiatoolbox`.
+
+To understand better how the programs work, study the jupyter notebooks referred to under the heading **Examples Taster**.
+
+### Command Line
+tiatoolbox supports various features through command line. For more information, please try `tiatoolbox --help`
```

Comment:
This doesn't appear to be related to tissue masking CLI but can;t see harm in it being in this PR.